File: mybatis-plus-join-test/test-join/src/main/java/com/github/yulichang/test/join/mapper/UserDTOMapper.java
package com.github.yulichang.test.join.mapper;

import com.github.yulichang.base.MPJBaseMapper;
import com.github.yulichang.test.join.entity.UserDto;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface UserDTOMapper extends MPJBaseMapper<UserDto> {
}

File: mybatis-plus-join-test/test-kotlin/src/main/java/com/github/yulichang/test/kt/mapper/UserDTOMapper.kt
package com.github.yulichang.test.kt.mapper

import com.github.yulichang.base.MPJBaseMapper
import com.github.yulichang.test.kt.entity.UserDto
import org.apache.ibatis.annotations.Mapper

@Mapper
interface UserDTOMapper : MPJBaseMapper<UserDto?>
